cmd/nerdctl/login.go

```diff
@@ -205,7 +205,12 @@ func ConfigureAuthentification(authConfig *types.AuthConfig, options *loginOptio
 	}
 
 	if options.username == "" {
-		return fmt.Errorf("error: Username is Required")
+
+		fmt.Print("Enter Username: ")
+		_, err := fmt.Scanf("%s", &options.username)
+		if err != nil {
+			return fmt.Errorf("error: Username is Required")
```

Comment:
You should return `err` here and then test if username is empty like we did for password https://github.com/containerd/nerdctl/blob/fcba3f1ce5f5d81f74447204103d118f93ecdce7/cmd/nerdctl/login.go#L227